lms/djangoapps/instructor_task/api_helper.py

```diff
@@ -255,10 +255,14 @@ def check_arguments_for_rescoring(usage_key):
     in).  An ItemNotFoundException is raised if the corresponding module
     descriptor doesn't exist.  NotImplementedError is raised if the
     corresponding module doesn't support rescoring calls.
+
+    Note: the string returned here is surfaced as the error
+    message on the instructor dashboard when a rescore is
+    submitted for a non-rescorable block.
     """
     descriptor = modulestore().get_item(usage_key)
     if not _supports_rescore(descriptor):
-        msg = "Specified module does not support rescoring."
+        msg = "This component cannot be rescored."
```

Comment:
Can you change this to `_("This component cannot be rescored")` so the error message can be translated?  The error message returned below in `check_entrance_exam_problems_for_rescoring` is translated.  Not sure why this was missed.